README.md

```diff
@@ -10,9 +10,16 @@ python -m http.server 8001
 The service is proxied off of `/proxy/` at https://address.of.notebook.server/user/theuser/proxy/8001.
 
 ## Installation
-Run
+To install:
 ```
 pip install git+https://github.com/jupyterhub/nbserverproxy
+```
+or 
+```
+conda install -c conda-forge nbserverproxy
+```
+Enable extension by running:
```

Comment:
I've seen other extensions that did not need enabling anymore when they were installed via conda. Do you know what needs to happen on the conda package side for that to be the case here as well?

Reply:
I'm working on enabling this on the conda side. Stay tuned. Thanks.